File: ProjectFolders/WebApi/WebApi/Models/PostClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApi.Models
{
    public class PostClass
    {
        public string username { get; set; }
        public string post_type { get; set; }
        public string pwd_hash { get; set; }
        public string post_title { get; set; }
        public int post_parrent { get; set; }
        public int hanesst_id { get; set; }
        public string post_text { get; set; }
    }
}

File: ProjectFolders/GoAPI - Core - Simulator/PostClass.go
package main


type PostRequest struct {
	Username string `json:"username"`
	Post_type string `json:"post_type"`
	Pwd_hash string `json:"pwd_hash"`
	Post_title string `json:"post_title"`
	Post_parrent int `json:"post_parrent"`
	Hanesst_id int `json:"hanesst_id"`
	Post_text string `json:"post_text"`
}

